app/javascript/packs/colorPreview.js

```diff
@@ -55,7 +50,7 @@ function initPreview() {
     textField.textColor.addEventListener('keyup', watchTextFields);
 
     // on init
-    updateColorFields();
+    updateFields(colorField, textField);
```

Comment:
the name should be something like `swapColorFields` because that's what you're ultimately doing in that function...